openslides_backend/action/actions/committee/import_meeting.py

```diff
@@ -0,0 +1,241 @@
+                if entry[field]:
+                    entry[field] = replace_map[target_collection][entry[field]]
+            elif isinstance(model_field, RelationListField):
+                target_collection = model_field.get_target_collection().collection
+                entry[field] = [
+                    replace_map[target_collection][id_] for id_ in entry[field]
+                ]
+            elif isinstance(model_field, GenericRelationField):
+                if entry[field]:
+                    name, id_ = entry[field].split(KEYSEPARATOR)
+                    entry[field] = (
+                        name + KEYSEPARATOR + str(replace_map[name][int(id_)])
+                    )
+            elif isinstance(model_field, GenericRelationListField):
+                new_fqid_list = []
+                for fqid in entry[field]:
+                    name, id_ = fqid.split(KEYSEPARATOR)
+                    new_fqid_list.append(
+                        name + KEYSEPARATOR + str(replace_map[name][int(id_)])
+                    )
+                entry[field] = new_fqid_list
+            if (
+                isinstance(model_field, BaseTemplateField)
+                and model_field.replacement_collection
+                and not model_field.is_template_field(field)
+            ):
+                replacement = model_field.get_replacement(field)
+                id_ = int(replacement)
+                new_id_ = replace_map[model_field.replacement_collection.collection][
+                    id_
+                ]
+                new_field = model_field.get_structured_field_name(new_id_)
+                tmp = entry[field]
+                del entry[field]
+                entry[new_field] = tmp
+
+    def create_write_requests(self, instance: Dict[str, Any]) -> Iterable[WriteRequest]:
+        json_data = instance["meeting_json"]
+        write_requests = []
+        for collection in json_data:
+            for entry in json_data[collection]:
+                fqid = FullQualifiedId(Collection(collection), entry["id"])
+                write_requests.append(
+                    self.build_write_request(
+                        EventType.Create,
+                        fqid,
+                        f"import meeting {json_data['meeting'][0]['id']}",
+                        entry,
+                    )
+                )
+        # add meeting to committee/meeting_ids
+        write_requests.append(
+            self.build_write_request(
+                EventType.Update,
+                FullQualifiedId(Collection("committee"), instance["id"]),
+                f"import meeting {json_data['meeting'][0]['id']}",
+                None,
+                {"add": {"meeting_ids": [json_data["meeting"][0]["id"]]}, "remove": {}},
+            )
+        )
+        return write_requests
+
+    def check_permissions(self, instance: Dict[str, Any]) -> None:
+        return
```

Comment:
Still TODO, right?

Reply:
Yes, that is TODO, What permissions should this action have?

Reply:
https://github.com/OpenSlides/OpenSlides/wiki/meeting.import#permissions

Reply:
Add the permission.